src/com/facebook/buck/apple/ProvisioningProfileCopyStep.java

```diff
@@ -192,20 +193,23 @@ public StepExecutionResult execute(ExecutionContext context)
     // Copy the actual .mobileprovision.
     filesystem.copy(provisioningProfileSource, provisioningProfileDestination, CopySourceMode.FILE);
 
+    // Create the application identifier from the app ID and bundle ID
+    String appID = bestProfile.get().getAppID().getFirst() + "." + bundleID;
```

Comment:
@rmaz To improve readability, I would actually write this out as:

```java
String appID = bestProfile.get().getAppID().getFirst() + "." + bundleID;
ImmutableMap<String, NSObject> overrideKeys = ImmutableMap.of(APPLICATION_IDENTIFIER, new NSString(appID));
```

Because the purpose of `ImmutableMap.of(APPLICATION_IDENTIFIER, new NSString(appID))` is not clear in the constructor of `PlistProcessStep`.

I'd also move this inside the `if` statement (`entitlementsPlist.isPresent()`) as it's only used inside that scope.